pkg/kotsutil/kots.go

```diff
@@ -564,6 +585,53 @@ func LoadKotsKindsFromPath(fromDir string) (*KotsKinds, error) {
 	return &kotsKinds, nil
 }
 
+func KotsKindsFromMap(kotsKindsMap map[string][]byte) (*KotsKinds, error) {
```

Comment:
is this only used for testing? i'd suggest writing the files to a directory and calling the `LoadKotsKindsFromPath` so that it uses the same workflow other callers do.